test/integration/features/F001_Valid_Change_Events.feature

```diff
@@ -41,3 +41,51 @@ Feature: F001. Ensure valid change events are converted and sent to DOS
     When the Changed Event is sent for processing with "valid" api key
     Then the processed Changed Request is sent to Dos
     And the Changed Request with changed address is captured by Dos
+
+@complete @kit
+  Scenario: F001S007. A valid change event with special characters is processed by DOS
```

Comment:
Can all these tests be marked with `@dev` as well (if they can run on dev/task environments)